app/src/androidTest/java/org/mozilla/fenix/ui/robots/BookmarksRobot.kt

```diff
@@ -0,0 +1,29 @@
+package org.mozilla.fenix.ui.robots
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withEffectiveVisibility
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import org.hamcrest.Matchers.allOf
+import org.mozilla.fenix.R
+
+class BookmarksRobot {
+
+    fun verifyBookmarksMenuView() = assertBookmarksView()
+
+    class Transition
```

Comment:
It looks like this empty Transition class should be removed (not doing anything).  Let me know if you have issues with this.

Reply:
This is used in LibraryRobot's openBookmarks() Transition method and then in bookmarksButtonTest().

Reply:
🤔 hmmm, I'm not sure it's a good practice to leave an empty transition class.  If the tests works with an empty transition class but fails when you remove it, perhaps we just need to rewrite the methods.